src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_TypeInference.cs

```diff
@@ -0,0 +1,1341 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
+{
+    public partial class StaticNullChecking : CSharpTestBase
+    {
+        // PROTOTYPE(NullableReferenceTypes): `default(string)` should be non-nullable string.
```

Comment:
The test already shows that `var` has a non-nullable type. Could you clarify the comment with expected behavior change?

Reply:
The test fails currently. It does not produce the expected warning.